retry/retry_test.go

```diff
@@ -181,4 +185,23 @@ func TestRetrier(t *testing.T) {
 		assert.EqualValues(t, err.(RetryError).Scope, MaxDurationScope)
 		assert.Equal(t, err.(RetryError).Err, context.DeadlineExceeded)
 	})
+
+	t.Run("If logging on attempt error is set, it should log the error", func(t *testing.T) {
+		log, hook := test.NewNullLogger()
+		ctx := logger.ToCtx(t.Context(), log)
+
+		retrier := New(
+			WithMaxAttempts(2),
+			WithWaitDuration(10*time.Millisecond),
+			WithLoggingOnAttemptError(logrus.ErrorLevel),
+		)
+
+		retrier.Do(ctx, func(ctx context.Context) error {
+			return errors.New("TestError")
+		})
+
+		assert.Len(t, hook.Entries, 2)
+		assert.Contains(t, hook.Entries[0].Message, "attempt failed")
```

Comment:
There is an error in the order of parameters, it should be `expectec, actual`.
```suggestion
		assert.Contains(t, "attempt failed", hook.Entries[0].Message)
```